java/src/jmri/jmrix/openlcb/OlcbProgrammerManager.java

```diff
@@ -43,15 +42,13 @@
  */
 public class OlcbProgrammerManager extends jmri.managers.DefaultProgrammerManager {
 
-    public OlcbProgrammerManager(Programmer pProgrammer) {
-        super(pProgrammer);
-    }
+    private final SystemConnectionMemo system;
 
-    public OlcbProgrammerManager(Programmer pProgrammer, jmri.SystemConnectionMemo memo) {
-        super(pProgrammer, memo);
+    public OlcbProgrammerManager(SystemConnectionMemo system) {
+        super(new OlcbProgrammer(system.get(OlcbInterface.class), null));
```

Comment:
Not sure, but I think the problem with NullPointerException is that you haven't registered the OlcbInterface instance in the SystemConnectionMemo instance so that `system.get(OlcbInterface.class)` returns null.